main.py

```diff
@@ -61,7 +63,9 @@ def __init__(self, json):
 
     url_data = json.load(urllib2.urlopen("http://"+ i + "/net_info"))
     foo = Info (url_data)
-    if len(foo.info) < 5:
+    if len(foo.info) == 1:
+        print colored("Insufficient nodes - under 5 nodes available!  Current count " + str(len(foo.info)) + " node", 'red')
+    if 1 < len(foo.info) < 5:
```

Comment:
Why not `if 1 <= len(foo.info) < 5:` or `len(foo.info) < 5:` and remove the `==1` case?